yamlExamples/MultipleStagesExample.md

```diff
@@ -0,0 +1,101 @@
+#### Jenkinsfile.yaml example with multiple stages
+
+```yaml
+stages:
+  - name: satge1
+    agent: any
+    
+    steps:
+      - sh: "scripts/hello"
+      - sh: "scripts/hello"
+      - sh:
+          script: "scripts/hello"
+          
+    post:
+      failure:
+        - sh: "scripts/hello"
+
+
+  - name: stage2
+    agent:
+      docketImage: maven:3-alpine
+      customWorkspace: path_to_workspace
+      
+    steps: 
+      - echo: "Hello from stage2"
+      
+      post:
+        success:
+          echo: "stage2 was a success"
+post:
+  always:
+    - sh: "Pipeline completed"
```

Comment:
no echo statement? Doesn't sh take the name of something it can execute?

Reply:
Yes it should be echo statement.